ParseLiveQuery/src/test/java/com/parse/TestParseLiveQueryClient.java

```diff
@@ -405,8 +405,25 @@ public void testCallbackNotifiedOnDisconnect() throws Exception {
         parseLiveQueryClient.registerListener(callbacks);
         callbacks.transcript.assertNoEventsSoFar();
 
+        // Unexpected close from the server:
         webSocketClientCallback.onClose();
-        callbacks.transcript.assertEventsSoFar("onLiveQueryClientDisconnected");
+        callbacks.transcript.assertEventsSoFar("onLiveQueryClientDisconnected: false");
+    }
+
+
+    @Test
+    public void testCallbackNotifiedOnDisconnect_expected() throws Exception {
```

Comment:
Hmm, ok - I've typically used `_` to delineate a related test with different expectations or states.

Looking at the Parse-SDK-Android repo though, I see that does not appear to be the case.  Will rename.